spec/features/user_creates_review_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+# [] Visiting the `/landmarks/1` path should show the landmark details for a landmark with the ID of 1.
+
+feature "user creates a new review" do
+  scenario "user views landmark and adds review" do
+    statue_of_liberty = Landmark.create(
+      name: "Statue of Liberty",
+      location: "New York",
+      image: "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a1/Statue_of_Liberty_7.jpg/500px-Statue_of_Liberty_7.jpg",
+      description: "A colossal neoclassical sculpture on Liberty Island"
+    )
+
+    user_login
+
+    visit landmarks_path
+
+    click_link "Statue of Liberty"
+
+    expect(page).to have_content statue_of_liberty.name
+
+    fill_in "Review", with: "It's SO GREEN, GUYS."
+    click_on "Submit"
+    expect(page).to have_content("It's SO GREEN, GUYS.")
+    expect(page).to have_content("herpderp")
+  end
```

Comment:
Please add a blank line here for readability. (And between all scenario blocks in this and your other feature test files)